hspy_bundle/configure_installer.py

```diff
@@ -193,6 +193,8 @@ def install_local_packages(self):
             else:
                 packages = glob(os.path.join(self.dist_path,
                                              "packages2install\\*amd64*"))
+            packages += glob(os.path.join(self.dist_path,
+                                         "packages2install\\*any*"))
```

Comment:
unless the names are separated by underscores instead... Sorry I merged this, I needed it for the release. We will be able to tweak it later anyway :)